cmd/nrtcacheck/main.go

```diff
@@ -116,7 +116,7 @@ func main() {
 
 	var nodeNames []string
 	if parsedArgs.NodeNames.Len() > 0 {
-		nodeNames = parsedArgs.NodeNames.List()
+		nodeNames = parsedArgs.NodeNames.UnsortedList()
```

Comment:
Use of `sets.List(parsedArgs.NodeNames)` seems more appropriate here. It has been documented here: https://pkg.go.dev/k8s.io/apimachinery/pkg/util/sets#List. Is there any particular reason we are going with [UnsortedList](https://pkg.go.dev/k8s.io/apimachinery/pkg/util/sets#Byte.UnsortedList) instead of List method provided in the sets package?

Also applies to similar updates below.

Reply:
Just thought order was not important in those cases so I used the UnsortedList function but your option seems better. 
Thanks for the comment.

Reply:
From the algorhythm perspective the order is not really relevant, actually I introduced room for bugs using `List` (vs `UnsortedList`. In this case it's better to preserve the old (and questionable) semantic than to sneak the change (I'd see fix) here.